Cooper/Controllers/GameReviewController.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Cooper.Models;
+
+// For more information on enabling Web API for empty projects, visit https://go.microsoft.com/fwlink/?LinkID=397860
+
+namespace Cooper.Controllers
+{
+    [Route("api/[controller]")]
+    public class GameReviewController : Controller
+    {
+        // GET: api/<controller>
+        [HttpGet]
+        public IEnumerable<GameReview> GetReviewsForGame()      // get users reviews for game
+        {
+            return new List<GameReview>();
+        }
+
+        [HttpGet]
+        public IEnumerable<GameReview> GetUserReviews()
+        {
+            return new List<GameReview>();      // get reviews for games from concrete user
+        }
+
+
+        // GET api/<controller>/5
+        [HttpGet("{id}")]
+        public GameReview Get(long id)
+        {
+            return new GameReview() { id = 10, Content = "Bad game" };
+        }
+
+        // POST api/<controller>
+        [HttpPost]
+        public IActionResult Post([FromBody]GameReview gameReview)
+        {
+            return Ok(new GameReview());
+        }
+
+        // PUT api/<controller>/5
+        [HttpPut("{id}")]
```

Comment:
Method is not needed. You should use the previous one (POST@api/review) and post the json with id